test/parallel/test-fs-read.js

```diff
@@ -53,3 +53,17 @@ test(Buffer.allocUnsafe(expected.length),
 test(new Uint8Array(expected.length),
      new Uint8Array(expected.length),
      Uint8Array.from(expected));
+
+{
+  // Reading beyond file length (3 in this case) should return no data.
+  // This is a test for a bug where reads > uin32 would return
+  // the start of the file.
```

Comment:
s/the start of the file/data from the current position in the file/

Reply:
it seems to be the start of the file

Reply:
nope you're right.